teachers_digital_platform/css/organisms/filter-panel.less

```diff
@@ -0,0 +1,48 @@
+// Filter panel
+
+.o-filter-panel {
+    .tdp-activity-search & {
+        background-color: #f7f8f9;
+        border: 1px solid #ccc;
+        padding: 30px;
+
+        .respond-to-min(900px , {
```

Comment:
Please replace pixel value with the `@bp-med-min` variable (and also on L15 and L23 below).

Reply:
Replaced.